docs/Esport Protocol V1/Tech integration/api-touchpoints.md

```diff
@@ -497,16 +509,12 @@ Upon successful completion of the withdrawal process, the server responds with a
 
 ### Historic Participation and Statistics
 
-#### Endpoint: GET `/api/v1/statistics/_historic?token={token}`
+#### Endpoint: GET `/fe/esports/statistics/historic`
 This endpoint provides historical data regarding a user's participation in gaming activities, specifically within the context of tournaments.
 
 ##### Request Header
 - `Content-Type`: application/json
-- `x-clientID`: [Your Client ID] – Your unique client identifier.
-- `x-clientSecret`: [Your Client Secret] – Your client secret for authentication.
-
-##### Query Parameters
-- `token`: The authentication token for the user session.
+- `user-jwt-token`: Users JWT token for authentication.
 
 ##### Response Structure
```

Comment:
@Jitendra-08 We have some more endpoints in esports route which is not covered here. Kindly check with @karthik-indigg / @JulianSharonIndigg regarding this and add those endpoints here if needed.
Missed endpoints:

1. /{tournamentUID}/leave
2. /statistics/summary